packages/utils/src/waitForEvent3.ts

```diff
@@ -32,13 +38,14 @@ const once = <T extends EventEmitter.ValidEventTypes>(
 export function waitForEvent3<T extends EventEmitter.ValidEventTypes>(
```

Comment:
Could add unit tests as these helper methods are now promoted to the `utils` package